macro/README.md

```diff
@@ -1,66 +1,22 @@
-# Iroha Macro
+# Iroha Macros
 
-## Description
+This crate contains macros and attributes for Iroha projects:
 
-This crate contains macros and attributes for Iroha projects.
-`iroha_derive` contains derive macros.
-
-### Features
-
-* `log` attribute for debugging functions inputs and output
+- `iroha_derive` is a custom derive macros for Iroha
 
 ## Usage
 
-### Requirements
-
-* [Rust](https://www.rust-lang.org/learn/get-started)
-
-### Build
-
-```bash
-cargo build
-```
-
-### Test
-
-```bash
-cargo test
-```
-
-### Add to your project
+Add the following to the manifest file of your Rust project:
 
 ```toml
 iroha_derive = { git = "https://github.com/hyperledger/iroha/", branch="iroha2-dev" }
 ```
 
-### Code example
+## Example
 
 ```rust
 #[derive(Clone, Debug, Encode, Decode)]
-pub struct Test [
-...
+pub struct Test {
+// ...
+}
 ```
```

Comment:
This example doesn't contain anything from this crate. I don't know why is it here originaly.
I'd suggest to replace it with example from [here](https://github.com/hyperledger/iroha/blob/61bbd74aec7c03da07df54d8f625a80fcf0231a0/macro/derive/src/lib.rs#L12)

Reply:
I've added the example, made a minor change in the comments there